src/scripts/main.js

```diff
@@ -1,3 +1,6 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.top = '175px';
+spider.style.left = '175px';
```

Comment:
what if the wall was 1000 px? as a programmer's we have to be prepared even for that